src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const listEl = document.querySelector('ul');
+
+const convertToNumber = str => Number(str.replace(/[^\d.-]/g, ''));
+
+const sortList = (list, sortParam) =>
+  [...list].sort(
+    (a, b) =>
+      convertToNumber(b.dataset[sortParam]) -
+      convertToNumber(a.dataset[sortParam]),
+  );
+
+const getEmployees = (sortedList, element) => {
+  element.innerHTML = '';
+
+  sortedList.forEach(item => {
+    const li = document.createElement('li');
+
+    li.setAttribute('data-position', item.dataset.position);
+    li.setAttribute('data-salary', item.dataset.salary);
+    li.setAttribute('data-age', item.dataset.age);
+
+    li.textContent = item.innerText;
+
+    element.appendChild(li);
+  });
+};
+
+const visibleEmployees = sortList([...listEl.children], 'salary');
```

Comment:
The `visibleEmployees` variable is not very clear. It seems to hold the sorted list of employees, but the name suggests that it might hold only the employees that are currently visible. Consider renaming this variable to something more descriptive, like `sortedEmployees`.